G1ANT.Addon.OCR.Tesseract/G1ANT.Addon.OCR.Tesseract/Commands/OcrOfflineFromScreenCommand.cs

```diff
@@ -21,37 +22,46 @@ public class OcrOfflineFromScreenCommand : Command
         public class Arguments : CommandArguments
         {
             [Argument(Required = true)]
-            public RectangleStructure Area { get; set; }
+            public RectangleStructure Area { get; set; } = new RectangleStructure(SystemInformation.VirtualScreen);
```

Comment:
Its shouldnt be required with default value

Reply:
done